doc/tutorials/basic/build_chatbot.md

```diff
@@ -0,0 +1,148 @@
+# Build a Chat Bot
+
+In this tutorial we will build a streaming *chat bot*. We will first use the *high-level* [`ChatInterface`](../../reference/chat/ChatInterface.ipynb) to build a basic chat bot. Then will will add streaming and finally we will make it scale to many users.
+
+:::{admonition} Note
+When I ask you to *run the code* in the sections below, you may either execute the code directly in the Panel docs via the green *run* button, in a cell in a notebook or in a file `app.py` that is served with `panel serve app.py --autoreload`.
+:::
+
+## Build a Basic Chat Bot
+
+Run the code below:
+
+```{pyodide}
+import panel as pn
+from time import sleep
+
+pn.extension()
+
+WIND_TURBINE = "A *wind turbine* is a device that converts the kinetic energy of wind into electrical energy."
+DONT_KNOW = "Sorry. I don't know."
+
+def get_response(contents, user, instance):
+    if "turbine" in contents.lower():
+        response = WIND_TURBINE
+    else:
+        response = DONT_KNOW
```

Comment:
Do we need to separate the response as constants above? Feels like that requires a tad more mental gymnastics to reference it above. Just a thought; I have no strong feelings for either way.
```suggestion
def get_response(contents, user, instance):
    if "turbine" in contents.lower():
        response = "A wind turbine converts wind energy into electricity."
    else:
        response = "Sorry, I don't know."
```